lib/node_modules/@stdlib/stats/base/dnanmean/README.md

```diff
@@ -180,6 +171,107 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dnanmean.h"
+```
+
+#### stdlib_strided_dnanmean( N, \*X, strideX )
+
+Calculate the arithmetic mean of a double-precision floating-point strided array `x`, ignoring `NaN` values.
+
+```c
+const double x[] = { 1.0, 2.0, 0.0/0.0, 3.0, 0.0/0.0, 4.0, 5.0, 6.0, 0.0/0.0, 7.0, 8.0, 0.0/0.0 };
+
+double v = stdlib_strided_dnanmean( 6, x, 2 );
+// returns 1.25
```

Comment:
@Neerajpathak07 Did you verify that this is the correct return value? 1+NaN+NaN+5.0+NaN+8 = 14, ignoring NaNs. 14/3 doesn't equal 1.25 according to my math.

Reply:
I see. It looks like you just copied the return value from the JS examples. This needs to be fixed, and likely also applies to your other PRs.